main.js

```diff
@@ -0,0 +1,65 @@
+var expenseCounter = function(num){
+  newNum = parseInt(num);
+  expenseTotal.textContent = newNum + count;
+  count = count + newNum;
```

Comment:
Do this line first, then you only need to assign `extenseTotal.textContent` to `count`